header-rewrite-filter/utility.cc

```diff
@@ -9,6 +9,8 @@ namespace Utility {
  OperationType StringToOperationType(absl::string_view operation) {
     if (operation == "set-header") {
         return OperationType::SetHeader;
+    } else if (operation == "append-header") {
```

Comment:
Nit: these strings would be good in constants. Ideally in the header, since the names of these commands might be useful to users outside this library.